arcgis-ios-sdk-samples/Utility network/Display content of utility network container/DisplayContentUtilityNetworkContainerViewController.swift

```diff
@@ -0,0 +1,360 @@
+    /// The map view managed by the view controller.
+    @IBOutlet var mapView: AGSMapView! {
+        didSet {
+            makeMap()
+            mapView.setViewpoint(AGSViewpoint(latitude: 41.801504, longitude: -88.163718, scale: 4e3))
+        }
+    }
+    /// The bar button item to prompt return of the main view.
+    @IBOutlet var exitBarButtonItem: UIBarButtonItem!
+    @IBOutlet var legendBarButtonItem: UIBarButtonItem!
+    
+    /// A feature service for an electric utility network in Naperville, Illinois.
+    let featureServiceURL = URL(string: "https://sampleserver7.arcgisonline.com/server/rest/services/UtilityNetwork/NapervilleElectric/FeatureServer")!
+    var utilityNetwork: AGSUtilityNetwork!
+    let graphicsOverlay = AGSGraphicsOverlay()
+    /// The default or previous viewpoint before entering the container view.
+    var previousViewpoint: AGSViewpoint?
+    /// An array containing information about the feature services' legends.
+    var legendInfos = [AGSLegendInfo]()
+    var featureLayers = [AGSFeatureLayer]()
+    
+    // The symbols used to display the container view contents.
+    let boundingBoxSymbol = ContainerViewSymbol(
+        name: "Bounding box",
+        symbol: AGSSimpleLineSymbol(style: .dash, color: .yellow, width: 3)
+    )
+    let attachmentSymbol = ContainerViewSymbol(
+        name: "Attachment",
+        symbol: AGSSimpleLineSymbol(style: .dot, color: .green, width: 3)
+    )
+    let connectivitySymbol = ContainerViewSymbol(
+        name: "Connectivity",
+        symbol: AGSSimpleLineSymbol(style: .dot, color: .red, width: 3)
+    )
+    
+    /// The data source for the legend table.
+    private var symbolsDataSource: SymbolsDataSource?
+    
+    /// The action that is prompted when exiting the container view.
+    @IBAction func exitContainerView() {
+        // Disable the bar button item since container view will be exited.
+        exitBarButtonItem.isEnabled = false
+        // Remove all the objects that were added onto the graphics overlay.
+        graphicsOverlay.graphics.removeAllObjects()
+        (mapView.map?.operationalLayers as? [AGSLayer])?.forEach { layer in
+            // Make each operational layer visible.
+            layer.isVisible = true
+        }
+        if let previousViewpoint = previousViewpoint {
+            // Return to the viewpoint before container view was entered.
+            mapView.setViewpoint(previousViewpoint) { [weak self] _ in
+                // Enable interaction on the map view.
+                self?.mapView.isUserInteractionEnabled = true
+            }
+        }
+    }
+    
+    /// Create a map.
+    ///
+    /// - Returns: An `AGSMap` object.
+    func makeMap() {
+        let webMapURL = URL(string: "https://sampleserver7.arcgisonline.com/portal/home/item.html?id=813eda749a9444e4a9d833a4db19e1c8")!
+        let map = AGSMap(url: webMapURL)
+        mapView.map = map
+    }
```

Comment:
I wanted to avoid force unwrapping `map`. I like the consistency of having the `makeMap()` function but I agree, it can be shortened – especially with a sample this long. I'll condense it to remove the function unless you think I should resolve it differently.